### DISCUSSION TITLE
Team template for process specific teams

### DISCUSSION THREAD

**Discussion by Dschoordsch** [ParabolInc/parabol#6246] [Ideas]
Some processes in Parabol are currently done by creating a temporary team and doing a number of predefined tasks. Examples include: on-boarding a member, interviewing a candidate, giving member feedback. It would be useful to have the ability to create short lived teams for these specific purposes. All these templates would need is a number of predefined tasks and these would be created and assigned to the team lead on creation.

Related:
* https://github.com/ParabolInc/parabol/discussions/6223

**Comment by avivapinchas** [comment]
I really like this idea! 

Making sure I understand: we would create these 'activity templates' or 'team templates', build them into Parabol, and maybe just have access to them ourselves, so we can optimize our own processes - is that right? 

> these would be created and assigned to the team lead on creation

For onboarding & feedback, most tasks need to be assigned to the team member (either new member or the member getting feedback) 

> interviewing a candidate

Do you mean conducting a BP? AFAIK, we use Notion in the earlier stages of interviewing, and I'd be hesitant to move away from that, since we can collaborate on interview notes there as well and have the custom stages for a candidate.

**Reply by Dschoordsch** [reply]
I think how you've described it is the best way to safely try it without too much work. It could be built into a feature offering if it proves to be useful, but then users would need to be able to create their own team templates.

> Do you mean conducting a BP? AFAIK, we use Notion in the earlier stages of interviewing, and I'd be hesitant to move away from that, since we can collaborate on interview notes there as well and have the custom stages for a candidate.

I mostly meant organizational tasks like:
* setting up pre-mortem
* create email address
* invite to slack
* create post-mortem
